exercises/run-length-encoding/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "run-length-encoding"
+version = "0.1.0"
+authors = ["zombiefungus <divmermarlav@gmail.com>"]
+
+[dependencies]
```

Comment:
just realised not all of our Cargo.toml have this line! Guess it's optional. The only exercise with a `[dependencies]` but no dependencies is acronym.

This line **is** included by `cargo new` though. So it makes sense to have.